components/Layout/Layout.jsx

```diff
@@ -1,10 +1,11 @@
 import Head from "next/head";
-import { Container, Flex, Text } from "@chakra-ui/react";
+import { Container, Flex, Text, Box } from "@chakra-ui/react";
+import { useRouter } from 'next/router'
```

Comment:
small nitpick :sweat_smile: 

```suggestion
import { useRouter } from 'next/router';
```

Reply:
Yes, we should settle on a linter perhaps. 
Will manually do these across the board now.